src/components/fields/schemaFields/SchemaField.test.tsx

```diff
@@ -443,4 +443,48 @@ describe("SchemaField", () => {
       await expectToggleOptions(container, expectedOptions);
     }
   );
+
+  test("don't render truthy root aware field", async () => {
+    const { container } = render(
+      <Formik
+        onSubmit={jest.fn()}
+        initialValues={{ apiVersion: "v3", config: { isRootAware: true } }}
+      >
+        <SchemaField
+          name="config.isRootAware"
+          schema={{
+            type: "boolean",
+          }}
+        />
+      </Formik>
+    );
+
+    // Renders no HTML element
+    expect(container.querySelector(".switch")).toBeNull();
+    await expectToggleOptions(container, []);
+  });
+
+  test.each([[undefined], [false]])(
```

Comment:
the following should work as well:
```suggestion
  test.each([undefined, false])(
```